packages/components/addon/components/hds/dropdown/index.hbs

```diff
@@ -14,7 +14,7 @@
     {{/if}}
   </:toggle>
   <:content>
-    <ul class="hds-dropdown-list hds-dropdown-list--absolute-right">
+    <ul class="hds-dropdown-list {{if (eq @listPosition "left") "hds-dropdown-list--position-left" "hds-dropdown-list--position-right"}}">
```

Comment:
[suggestion] this is even more compact (and easy to read, and future-proof).

```suggestion
    <ul class="hds-dropdown-list hds-dropdown-list--position-{{@listPosition}}">
```

Reply:
This would mean that it always needs to be explicitly defined. If not defined, it would be `hds-dropdown-list--postiion-` 

It would also create the same issue as the `is-{{state}}` classes created (since fixed).

Reply:
Created issue: https://github.com/hashicorp/design-system/issues/173